signer/tests/integration/transaction_coordinator.rs

```diff
@@ -1143,7 +1143,6 @@ async fn run_dkg_if_signatures_required_changes(change_signatures_required: bool
 
     // We want current signer set be same as bootstrap_signer_set to prevent DKG trigger
     // because of changed signer set. Also we want this signer set be at least 2 signers
```

Comment:
nit: orphaned comment

Reply:
Removed in https://github.com/stacks-sbtc/sbtc/pull/1707/commits/4fa16e0f1e35bdfb0b8c73c55524b5436942cce3